pkg/cli/check_diff.go

```diff
@@ -0,0 +1,35 @@
+package cli
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/wolfi-dev/wolfictl/pkg/checks"
+)
+
+func Diff() *cobra.Command {
+	o := checks.NewDiff()
+	cmd := &cobra.Command{
+		Use:               "diff",
+		DisableAutoGenTag: true,
+		SilenceUsage:      true,
+		SilenceErrors:     true,
+		Short:             "Create a diff comparing proposed apk changes to the latest available in an APKINDEX",
```

Comment:
Is this command expected to be run only after a melange build? That's probably worth calling attention to here 🤔

Reply:
thanks done